lib/views/alaveteli_pro/batch_request_authority_searches/_begginer_batch_pop_up.html.erb

```diff
@@ -36,7 +36,7 @@
         <button class="button js-carousel-button-next" aria-label="Next slide">
           Next →
         </button>
-        <button class="modal-close" aria-label="close modal">×</button>
+        <button class="modal-close" aria-label="close modal">Close</button>
```

Comment:
I think we should call this "Continue" here, as it more clearly implies they can continue with their batch. "Close" makes it sound like it might close the batch interface.